### ISSUE TITLE
FEATURE: add a welcome intro at the beginning

### ISSUE THREAD

**Issue by AnushkaS77** [YadavAkhileshh/Alien-Invasion-Defense#604]
### Feature Summary

hello, before the main homepage appears i would like to add an welcome to intro to make it more creative....ive already done the code part...only after the issue is assigned and tagged with labels ill change the intro video suitable for this project...
ive worked on previous projects as well with this same feature update...kindly assign me this issue with gssoc-ext, hackoctober-fest and label tag thankyou 

### Description

[Alien Invasion Defense.webm](https://github.com/user-attachments/assets/67e1eb33-1a3c-4353-beaa-cf98dd833285)

### Additional Information

- [X] I have searched for existing feature requests
- [X] I am willing to help implement this feature
- [X] I can provide more details or clarification if needed

**Comment by AnushkaS77** [comment]
@YadavAkhileshh  ive raised pr...kindly check it
